src/Aspire.Cli/Commands/NewCommand.cs

```diff
@@ -68,14 +68,14 @@ public NewCommand(IDotNetCliRunner runner, INuGetPackageCache nuGetPackageCache,
         //       Once we integrate with template engine we will also be able to
         //       interrogate the various options and add them. For now we will 
         //       keep it simple.
-        Dictionary<string, (string TemplateName, string TemplateDescription, string? PathAppendage)> validTemplates = new(StringComparer.OrdinalIgnoreCase) {
-            { "aspire-starter", ("aspire-starter", "Aspire Starter App", "./src")},
-            { "aspire", ("aspire", "Aspire Empty App", "./src") },
-            { "aspire-apphost", ("aspire-apphost", "Aspire App Host", "./") },
-            { "aspire-servicedefaults", ("aspire-servicedefaults", "Aspire Service Defaults", "./") },
-            { "aspire-mstest", ("aspire-mstest", "Aspire Test Project (MSTest)", "./") },
-            { "aspire-nunit", ("aspire-nunit", "Aspire Test Project (NUnit)", "./") },
-            { "aspire-xunit", ("aspire-xunit", "Aspire Test Project (xUnit)", "./")}
+        Dictionary<string, (string TemplateName, string TemplateDescription, Func<string, string> PathDeriver)> validTemplates = new(StringComparer.OrdinalIgnoreCase) {
+            { "aspire-starter", ("aspire-starter", "Aspire Starter App", projectName => $"./{projectName}")},
+            { "aspire", ("aspire", "Aspire Empty App", projectName => $"./{projectName}") },
+            { "aspire-apphost", ("aspire-apphost", "Aspire App Host", _ => "./") },
+            { "aspire-servicedefaults", ("aspire-servicedefaults", "Aspire Service Defaults", _ => "./") },
+            { "aspire-mstest", ("aspire-mstest", "Aspire Test Project (MSTest)", _ => "./") },
+            { "aspire-nunit", ("aspire-nunit", "Aspire Test Project (NUnit)", _ => "./") },
+            { "aspire-xunit", ("aspire-xunit", "Aspire Test Project (xUnit)", _ => "./")}
```

Comment:
Why are these different?

Reply:
Those are more "project" templates rather than solution templates. So you would probably be in the existing solution folder when you create them so the expected path would be different (you aren't creating an intermediate "solution" folder.